src/raster/buffer.rs

```diff
@@ -220,7 +220,12 @@ impl<T: GdalType + Copy> From<Array2<T>> for Buffer<T> {
         let shape = value.shape();
         let (cols, rows) = (shape[1], shape[0]);
         let data: Vec<T> = if value.is_standard_layout() {
-            value.into_raw_vec()
+            let (data, offset) = value.into_raw_vec_and_offset();
+            if let Some(offset) = offset {
+                data.into_iter().skip(offset).collect()
+            } else {
+                data
```

Comment:
Let's also handle `offset == 0` the same way.

Reply:
Done - I removed the if

Reply:
That might do an extra copy in the zero offset case.